readme.md

```diff
@@ -185,6 +185,17 @@ $ npm install --save-dev branchsite
 
 For new packages, start the `version` field in package.json at `0.0.0` and let `np` bump it to `1.0.0` or `0.1.0` when publishing.
 
+### Release an update to an old major version
+
+To release a minor/patch version for an old major version, create a branch from the major version's git tag and run `np`:
+
+```bash
+$ git checkout -b fix-old-bug v1.0.0 # (1.0.0 being the previous major version)
+# Create some commits....
+$ git push --set-upstream origin fix-old-bug
```

Comment:
I think you can just do `git push --set-upstream origin HEAD` and git will use the same branch name as used locally.